pubsub/README.rst

```diff
@@ -65,102 +69,96 @@ Here's the rough usage pattern for subscribing:
 Configuration
 ^^^^^^^^^^^^^
 
-Our create_subscription method is a thing wrapper and thus supports all keyword
-configuration arguments from the official pubsub client which you can find in
-the `official Google documentation`_.
+Our create_subscription method is a thin wrapper and thus supports almost all
+keyword configuration arguments from the official pubsub client which you can
+find in the `official Google documentation`_.
+
+Since the underlying Google implementation of ``Scheduler`` only allows for the
+concrete ``ThreadScheduler`` which is also the default, we have opted not to
+expose this configuration option. Additionally, we would like to fully
+deprecate said Google implementation in favour of a fully `asyncio`
```

Comment:
two backticks around `asyncio` (one backtick implies link/ref, two backticks is shorthand for `:code:`)

Reply:
Oops, this was just force of habit in Slack! Good catch.